docs/notifications.json

```
@@ -18,5 +13,10 @@
     "id": 79,
     "title": "<b>Influence the roadmap for 2024</b>",
```

While I'm at it https://github.com/mui/material-ui/pull/40552#issuecomment-1906941841 😄

```suggestion
    "title": "<b>A new Developer Survey is open</b>",
```